### PR TITLE
Grouping some scattered districts into group by postal areas

### PR DESCRIPTION
**Pull request by gwentruong**

### CODE REVIEW

File: data/metropolitan.prj [hunk 1]
```diff
@@ -0,0 +1 @@
+GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]
```
Comment (Diem-Nguyen) [inline data/metropolitan.prj]: dont understand this step that much :D. Why we need to add?

File: data/small_district_transaction.csv [hunk 2]
```diff
@@ -0,0 +1,70 @@
+id;posti_alue;nimi;price_m2_mean
```
Comment (Diem-Nguyen) [inline data/small_district_transaction.csv]: do we have any counts for it, apart from mean price?

File: data/metropolitan.prj (hunk 1 above)
Comment (gwentruong) [reply data/metropolitan.prj]: This file is auto generated to support shp file.

File: data/small_district_transaction.csv (hunk 2 above)
Comment (gwentruong) [reply data/small_district_transaction.csv]: I can add counts if it is needed. What would you use the count for?

Comment (ritskinguyen) [reply data/small_district_transaction.csv]: We would need the counts to evaluate the significant of a district

**Pull request merged**